assets/i18n/de.toml

```
@@ -182,6 +182,12 @@ loadpoint = "Ladepunkt"
 vehicle = "Fahrzeug"
 energy = "Geladen"
 date = "Beendet"
+identifier = "Kennung" # csv
+chargedenergy = "Energie (kWh)" # csv
+meterstart = "Anfangszählerstand (kWh)" # csv
+meterstop = "Endzählerstand (kWh)" # csv
+created = "Startzeit" # csv
+finished = "Endzeit" # csv
```

```suggestion

[sessions.csv]
identifier = "Kennung"
chargedenergy = "Energie (kWh)"
meterstart = "Anfangszählerstand (kWh)"
meterstop = "Endzählerstand (kWh)"
created = "Startzeit"
finished = "Endzeit"
```

Lieber ne eigene Untergruppe als jeweils Kommentare

---

Dann müsste ich das Fahrzeug z.b. doppeln. Ist das wünschenswert?

---

Ja gerade bei Translation würde ich immer doppeln. Suchen/Ersetzen ist in dieser Datei total einfach und man hat keine Seiteneffekt weil ein Text an einer Stelle eingesetzt wird wo man ihn nicht erwartet hätte.